packages/test/test-end-to-end-tests/src/test/gc/gcContainerRuntimeCompat.spec.ts

```diff
@@ -181,7 +181,8 @@ describeFullCompat("GC summary compatibility tests", (getTestObjectProvider) =>
          * This test validates that the unreferenced timestamp in the summary generated by a container runtime can
          * be read by older / newer versions of the container runtime.
          */
-        it(`runtime version ${v2Str} validates unreferenced timestamp from summary by version ${v1Str}` , async () => {
+        // Issue #10053
+        it.skip(`runtime version ${v2Str} validates unreferenced timestamp from summary by version ${v1Str}` , async () => {
```

Comment:
You should skip the whole describeFullCompat block. That way nothing will run in this file.